algorithms.tex

```diff
@@ -1,2 +1,17 @@
+% A description of some of the algorithms in SymPy. The list is not
```

Comment:
Maybe. We would have to rethink how the feature list looks like. Right now it's just a section for each of the most core features, and then a short listing of the rest of the features. 

I'm actually most unsure about this algorithms section to be honest.